query/stdlib/influxdata/influxdb/operators.go

```diff
@@ -0,0 +1,55 @@
+package influxdb
+
+import (
+	"fmt"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/values"
+)
+
+const ReadRangePhysKind = "ReadRangePhysKind"
+
+type ReadRangePhysSpec struct {
```

Comment:
This is a divergence from how we've named the other `ProcedureSpec`s.  I think I like it though.  If there is a 1:1 correspondence with physical specs and RPCs, maybe it should match the RPC name as close as possible?

Reply:
Perhaps change `ReadRangePhysSpec` to just `ReadRange`? I wouldn't mind that.